floor_generator/lib/writer/query_method_writer.dart

```diff
@@ -137,11 +138,15 @@ class QueryMethodWriter implements Writer {
     @nullable final String arguments,
     @nonNull final String mapper,
   ) {
-    final entityName = _queryMethod.queryable.name;
-
+    final tableName = _queryMethod.queryable.name;
```

Comment:
💡Queryable names don't exclusively define table names but also views. What do you think about the following?

```suggestion
    final queryableName = _queryMethod.queryable.name;
```